src/go/k8s/tests/e2e/create-topic-given-cert-secret/01-redpanda-cluster.yaml

```diff
@@ -24,6 +24,13 @@ spec:
         nodeSecretRef:
           name: cluster-tls-node-certificate
           namespace: given-cert
+    schemaRegistry:
+      port: 8081
+      tls:
+        enabled: true
+        nodeSecretRef:
+          name: cluster-tls-node-certificate
+          namespace: given-cert
```

Comment:
Could you enable this in:

https://github.com/vectorizedio/redpanda/blob/315fdd75820f7ff7d4161a3156645ded3b6a1dc7/src/go/k8s/tests/e2e/create-topic-given-cm-secret-client-auth/01-redpanda-cluster.yaml#L17-L27

https://github.com/vectorizedio/redpanda/blob/315fdd75820f7ff7d4161a3156645ded3b6a1dc7/src/go/k8s/tests/e2e/create-topic-given-issuer/01-redpanda-cluster.yaml#L19-L28

Reply:
There is no assertion

Reply:
I'm not sure what this e2e test will verify. From the yaml files I see create topic. Can you add schema registry test?

Reply:
> is already in there which is Redpanda running

This is not enough in my opinion.

Reply:
I've added more tests specific to schema registry